metadata-ingestion/src/datahub/ingestion/source/looker/view_upstream.py

```diff
@@ -197,10 +200,20 @@ def _generate_fully_qualified_name(
     return sql_table_name.lower()
 
 
+@lru_cache(maxsize=128)
+def _get_schema_info(
```

Comment:
remove this - the schema resolver automatically caches things internally

Reply:
done